lib/opensusebasetest.pm

```diff
@@ -318,6 +318,9 @@ sub wait_boot {
     my $in_grub         = $args{in_grub} // 0;
     my $nologin         = $args{nologin};
 
+    # Our Hyper-V host is slow
+    $ready_time *= 2 if check_var('VIRSH_VMM_FAMILY', 'hyperv');
```

Comment:
I guess we have that covered in the meantime by setting TIMEOUT_SCALE=3 on the machine, don't you think?